iterators.tex

```diff
@@ -3929,8 +3929,10 @@
     common_iterator& operator=(const common_iterator<ConvertibleTo<I>, ConvertibleTo<S>>& u);
 
     @\seebelow@ operator*();
-    @\seebelow@ operator*() const;
-    @\seebelow@ operator->() const requires Readable<I>;
+    @\seebelow@ operator*() const
```

Comment:
Yes. I think our guidelines were to use "see below" for long/complicated/metaprogrammed return types, and simply use `decltype(auto)` directly. Fixing.